bot/seasons/evergreen/space.py

```diff
@@ -0,0 +1,154 @@
+import logging
+from datetime import datetime
+from random import randint
+from typing import Any, Dict, Optional
+from urllib.parse import urlencode
+
+from discord import Embed
+from discord.ext.commands import Cog, Context, command
+
+from bot.bot import SeasonalBot
+from bot.constants import Tokens
+
+logger = logging.getLogger(__name__)
+
+# NASA API base URL
+BASE_URL = "https://api.nasa.gov"
+NASA_IMAGES_BASE = "https://images-api.nasa.gov"
+EPIC_BASE_URL = "https://epic.gsfc.nasa.gov"
+
+# Default Parameters:
+# .apod command default request parameters
+APOD_PARAMS = {
```

Comment:
The comments don't really give additional context to the constant and, if you add `DEFAULT` to the constant's name, it becomes entirely self-documenting. 

```suggestion
APOD_DEFAULT_PARAMS = {
```